### PR TITLE
Mock pubmed 557

### PR DESCRIPTION
**Pull request by knirirr**
This mocks two tests as described in #557

### CODE REVIEW

**Comment by knirirr** [comment]
@proccaserra there are some test failures here and locally in other parts of the code; IIRC these are ones we have discussed before where counts have changed, e.g.

```
======================================================================
FAIL: test_b_ii_s_3 (isatab.validate.test_core.TestValidators.test_b_ii_s_3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/isa-api/tests/isatab/validate/test_core.py", line 20, in test_b_ii_s_3
    self.assertEqual(len(r['warnings']), 12)
AssertionError: 5 != 12
```

File: tests/utils/test_isatools_utils.py [hunk 1]
```diff
@@ -82,11 +83,12 @@ def test_get_ontology(self):
         ontology_source = ols.get_ols_ontology('efo')
         self.assertIsInstance(ontology_source, OntologySource)
         self.assertEqual(ontology_source.name, 'efo')
-        self.assertIn("https://www.ebi.ac.uk/ols", ontology_source.file)
-        self.assertIn("/api/ontologies/efo?lang=en", ontology_source.file)
-        self.assertIsInstance(ontology_source.version, str)
         self.assertEqual(
```
Comment (proccaserra) [inline tests/utils/test_isatools_utils.py:86]: @knirirr I recall that we coded it this way with @terazus to guard against version change, ( which makes the assumption that the service would keep the pattern and can not be guaranteed)

**Review by proccaserra — COMMENTED** [review]
apart from the comment on ols , the rest looks fine to me. thx @knirirr

**Comment by knirirr** [comment]
Thanks. I'm not clear on how the OLS part should look, I'm afraid.
I'm also assuming that the 'develop' branch is being used as this appears to be ahead of master. I've merged that into my branch and changed this PR to develop instead of master.

**Pull request merged**